File: MODUL 7/Soal 3/PRAK603-2210817310008-Muh-Haekal-Barera.py
baris1, baris2 = map(int, input().split())
if (baris1 !=baris2):
    print("Jumlah tidak sama")
else :
    x = list(map(int, input().split()))
    y = list(map(int, input().split()))
    print("\n")
    for i in range (baris1):
        print(x[i]*y[i], end=' ')

File: MODUL 7/Soal 3/PRAK603-2210817310008-Muh-Haekal-Barera.c
#include <stdio.h>

int main()
{
    int x, isi, baris1, baris2, hasil;
    scanf("%d %d", &baris1, &baris2);
    int bil1[baris1], bil2[baris2];
    if (baris1 == baris2)
    {
        for (x = 0; x < baris1; x++)
        {
            scanf("%d", &isi);
            bil1[x] = isi;
        }
        for (x = 0; x < baris2; x++)
        {
            scanf("%d", &isi);
            bil2[x] = isi;
        }
        printf("\n");
        for (x = 0; x < baris1; x++)
        {
            hasil = bil1[x] * bil2[x];
            printf("%d", hasil);
            printf(" ", hasil);
        }
    }
    else
    {
        printf("\nJumlah tidak sama");
    }
}